FireMotD

```diff
@@ -205,6 +205,12 @@ GatherInfo () {
             Platform="Xen Hypervisor v""$(echo "$Xen" | sed -n -e 's/^.*Xen version: //p')"
     elif [[ "$Rasp" = *"Rasp"* ]] ; then
             Platform=${Rasp//*: /}
+    elif [[ $(uname -r) =~ "pine64-longsleep" ]] ; then
+            Platform="Pine A64"
+            _mem="$(< /proc/meminfo grep MemTotal | awk '{print $2}')"
+            if [ $_mem -gt  500000 ] && [ $_mem -gt  503000 ]; then Platform="Pine A64 512MB Rev B Board"; fi
```

Comment:
I think this should be 

if [ $_mem -gt  500000 ] && [ $_mem -**lt**  503000 ]         

same on other lines

Reply:
@leonsio I was surprised to see this mistake too. See #96